Modelica/Electrical/Analog/Examples/Lines/PowerLineWithFence.mo

```diff
@@ -0,0 +1,132 @@
+  extends Modelica.Icons.Example;
+  Modelica.Electrical.Analog.Lines.M_OLine line(
+    N=2,  c = Ccomp, g = fill(1e-12, div(g.n * (g.n + 1), 2)), l = Lcomp,
+    length=100e3,  lines = g.n, r = g.R1) annotation (
+    Placement(visible = true, transformation(origin={-28,22},     extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Electrical.Analog.Sources.SineVoltage v2(
+    V=v1.V,
+    phase=-2.0943951023932,
+    f=v1.f) annotation (Placement(visible=true, transformation(
+        origin={-62,22},
+        extent={{10,10},{-10,-10}},
+        rotation=180)));
+  Modelica.Electrical.Analog.Basic.Resistor rgUL(R = 1) annotation (
+    Placement(visible = true, transformation(origin={-86,-28},    extent = {{-8, 8}, {8, -8}}, rotation = 270)));
+  Modelica.Electrical.Analog.Basic.Ground ground annotation (
+    Placement(visible = true, transformation(extent={{-96,-60},{-76,-40}},      rotation = 0)));
+  Modelica.Electrical.Analog.Sources.SineVoltage v3(
+    V=v1.V,
+    phase=2.0943951023932,
+    f=v1.f) annotation (Placement(visible=true, transformation(
+        origin={-62,2},
+        extent={{10,10},{-10,-10}},
+        rotation=180)));
+  Modelica.Electrical.Analog.Basic.Resistor r1(R=500) annotation (Placement(
+        visible=true, transformation(
+        origin={10,38},
+        extent={{-8,-8},{8,8}},
+        rotation=0)));
+  Modelica.Electrical.Analog.Basic.Resistor r2(R=r1.R) annotation (Placement(
+        visible=true, transformation(
+        origin={10,22},
+        extent={{-8,-8},{8,8}},
+        rotation=0)));
+  Modelica.Electrical.Analog.Basic.Resistor r3(R=r1.R) annotation (Placement(
+        visible=true, transformation(
+        origin={10,2},
+        extent={{-8,-8},{8,8}},
+        rotation=0)));
+  parameter Modelica.Electrical.Analog.Lines.Functions.LineGeometry g(
+    n=4,
+    x={0,-3.048,3.048,-9.144},
+    y={12.192,12.192,12.192,3.048},
+    r=1e-3/2*{12.7,12.7,12.7,4.064},
+    R1=1e-3*{0.348,0.348,0.348,1.802},
+    k_s={0.287,0.287,0.287,0.779},
+    f=60)
+    annotation (Placement(transformation(extent={{42,10},{62,30}})));
+  Modelica.Electrical.Analog.Sources.SineVoltage v1(V=345e3*sqrt(2/3), f=60)
+    annotation (Placement(visible=true, transformation(
+        origin={-62,40},
+        extent={{10,10},{-10,-10}},
+        rotation=180)));
+  Modelica.Electrical.Analog.Basic.Resistor body(R=1000) annotation (
+      Placement(visible=true, transformation(
+        origin={-18,-18},
+        extent={{-8,8},{8,-8}},
+        rotation=270)));
+  parameter Real Rcomp[div(g.n * (g.n + 1), 2)](each final unit="Ohm/m", each fixed=false) "Compact resistance matrix";
+  parameter Real Xcomp[div(g.n * (g.n + 1), 2)](each final unit="Ohm/m", each fixed=false) "Compact reactance matrix";
+  parameter Real Lcomp[div(g.n * (g.n + 1), 2)](each final unit="H/m",  each fixed=false) "Compact inductance";
+  parameter Real Ccomp[div(g.n * (g.n + 1), 2)](each final unit="F/m")=
```

Comment:
It is common for lines to use parameters per unit length.
This is unavoidable, since actual inductances and capacitanses of lines are proportional to line lengths.
The units proposed here are SI, even though do not appear in Modelica.Units.SI.